ckanext/lacounts/harvest/harvesters/socrata.py

```diff
@@ -1,13 +1,24 @@
-import logging
-from ckanext.harvest.harvesters import CKANHarvester
+import json
+
+from ckanext.socrata.plugin import SocrataHarvester
 from ckanext.lacounts.harvest import helpers
-log = logging.getLogger(__name__)
 
 
-#TODO: switch to SocrataHarvester
-class LacountsSocrataHarvester(CKANHarvester):
+class LacountsSocrataHarvester(SocrataHarvester):
+
+    def import_stage(self, harvest_object):
+        try:
+            package = json.loads(harvest_object.content)
+        except TypeError:
+            # harvest_object may not have content. For example, if set for
+            # deletion.
+            pass
+        else:
+            package = helpers.process_package(package, harvest_object)
+            harvest_object.content = json.dumps(package)
+        return super(LacountsSocrataHarvester, self) \
+            .import_stage(harvest_object)
 
     def process_package(self, package, harvest_object):
```

Comment:
So this method will be called by `ckanext.socrata.plugin.SocrataHarvester` at some point of `import_stage`? I can't find it for now though.